crates/resolver-tests/tests/resolve.rs

```diff
@@ -937,6 +937,46 @@ fn large_conflict_cache() {
     let _ = resolve(root_deps, &reg);
 }
 
+#[test]
+fn resolving_worst_case() {
+    let mut reg = Vec::new();
+
+    // increase in resolve time is at least cubic over `N`
```

Comment:
Do we have a regression fit demonstrating that it is technically `cubic`? I'd love to see it, or help make it. If not perhaps we should be less specific year "the resolver gets rapidly slower as you increase N".

Reply:
Yes, I edited the PR description.